ush/ioda/bufr2ioda/bufr2ioda_tesac_mammals_profiles.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env python3
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
```

Comment:
I'll only comment once but consider changing these to 2024

Reply:
Done.